docs/security.rst

```diff
@@ -16,9 +16,12 @@ Web Authentication
 Password
 ''''''''
 
+.. note:: This is for older version of the web UI only. If you are using FAB-based web UI with RBAC feature,
+   please use command line interface `create_user` to create accounts, or do that in the FAB-based UI itself.
```

Comment:
For multiline `note`:
```
.. note::

   This is for older version of the web UI only. If you are using FAB-based web UI with RBAC feature,
   please use command line interface `create_user` to create accounts, or do that in the FAB-based UI itself.
```

Reply:
Instead of `older version of web UI`, I would call it `flask-admin based Web UI`